src/sentry/static/sentry/app/views/settings/settingsLayout.jsx

```diff
@@ -54,18 +40,10 @@ let NewSettingsWarning = ({location = {}}) => {
   };
   let Component = isRouter ? Link : 'a';
```

Comment:
i know this is old, but this is a confusing and weird name. Can we call it something like `LinkWithFallback` or something?